objects/src/errors.rs

```diff
@@ -63,6 +66,8 @@ pub enum AccountDeltaError {
     TooManyRemovedAssets { actual: usize, max: usize },
     TooManyUpdatedStorageItems { actual: usize, max: usize },
     DuplicateStorageMapLeaf { key: RpoDigest },
+    AmountTooBig(u64),
```

Comment:
nit: I would probably make the naming a bit more explicit - e.g., `AssetAmountTooBig`.